src/components/transform-object-button.js

```diff
@@ -279,7 +279,7 @@ AFRAME.registerSystem("transform-selected-object", {
       .applyQuaternion(q.copy(plane.quaternion).invert())
       .multiplyScalar(SENSITIVITY / cameraToPlaneDistance);
     if (this.mode === TRANSFORM_MODE.CURSOR) {
-      const modify = AFRAME.scenes[0].systems.userinput.get(paths.actions.transformModifier);
+      const modify = !AFRAME.scenes[0].systems.userinput.get(paths.actions.transformModifier);
```

Comment:
This changes the default for object rotation to unsapped. This was useful for the camera tool, and is something we have said we should do for a long time, so I decided not to make it conditional.